verl/trainer/config/ppo_trainer.yaml

```diff
@@ -142,6 +142,7 @@ reward_model:
   ulysses_sequence_parallel_size: 1 # sp size
   use_dynamic_bsz: ${critic.use_dynamic_bsz}
   forward_max_token_len_per_gpu: ${critic.forward_max_token_len_per_gpu}
+  reward_manager: naive
```

Comment:
could you add this option to https://github.com/volcengine/verl/blob/main/docs/examples/config.rst as well? thanks!